browser/resources/settings/brave_search_engines_page/brave_search_engines_page.html

```diff
@@ -71,7 +71,7 @@
       sub-label="$i18n{searchSuggestDesc}">
   </settings-toggle-button>
 </template>
-<template is="dom-if" if="[[!prefs.brave.web_discovery_disabled_by_policy.value]]">
```

Comment:
As discussed on Slack, I think we can just do:
`prefs.brave.web_discovery_enabled.value`

Reply:
Circling back on this - the one we could have potentially done is:
```
<template is="dom-if" if="[[!prefs.brave.web_discovery_enabled_by_policy.userControlDisabled]]">
```

That works for IsManaged... but we're updating our checks to ALSO check for `BraveOriginState::GetInstance()->IsBraveOriginUser();`

I do introduce a variable for this in https://github.com/brave/brave-core/pull/30360. We could rework this check in the Origin settings screen PR.